klaus/__init__.py

```diff
@@ -17,7 +17,12 @@ class Klaus(flask.Flask):
 
     def __init__(self, repo_paths, site_name, use_smarthttp, ctags_policy='none'):
         """(See `make_app` for parameter descriptions.)"""
-        repo_objs = [FancyRepo(path) for path in repo_paths]
+
+        user_dirs = repo_paths
+        repo_objs = []
```

Comment:
These changes seem unrelated

Reply:
Sorry! These changes are for my self usage. Didn't notice these commits will be related to this pull request. I've checkout a new branch and made a new pull request. It was my first time doing pull request, don't know if it's the right way 😃

Reply:
I'll move your comments to the new pull request

Reply:
#211 new request, I've move some of your comments to there